test/compiler.test.js

```diff
@@ -706,6 +707,47 @@ describe('compiler', function() {
       instructions = boot.compile(appdir.PATH);
       expect(instructions.config).to.not.have.property('modified');
     });
+
+    it('has a mixinSources option', function() {
+      var options = { appRootDir: SIMPLE_APP, mixinSources: [MIXIN_SOURCES] };
+      var instructions = boot.compile(options);
+      expect(instructions.mixins).to.not.have.property('TimeStamps');
+      expect(instructions.mixins).to.have.property('Other');
```

Comment:
This test is difficult to understand on its own. Why `TimeStamps` should not exist and `Other` should exist?

Please use the helpers `appdir.createConfigFilesSync()`, `appdir.writeConfigFileSync()` and `appdir.writeFileSync()` to build the test scenario from within the test.